doc/reference/tooling/tt_cli/export.rst

```diff
@@ -0,0 +1,145 @@
+    :class: fact
+
+    This command is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+..  code-block:: console
+
+    $ tt [crud] export URI FILE SPACE [EXPORT_OPTION ...]
+
+``tt [crud] export`` exports a space's data to a file.
+The ``crud`` command is optional and can be used to export a cluster's data by using the `CRUD <https://github.com/tarantool/crud>`_ module. Without ``crud``, data is exported using the :ref:`box.space <box_space>` API.
+
+``tt [crud] export`` takes the following arguments:
+
+*   ``URI``: The URI of a router instance if ``crud`` is used. Otherwise, it should specify the URI of a storage.
+*   ``FILE``: The name of a file for storing exported data.
+*   ``SPACE``: The name of a space from which data is exported.
+
+..  NOTE::
+
+    You should have :ref:`read access <authentication-owners_privileges>` to the required space to be able to export data.
+
+..  NOTE::
+
+    Exporting isn't supported for the :ref:`interval <index-box_interval>` field type.
+
+
+.. _tt-export-default:
+
+Exporting with default settings
+-------------------------------
+
+The command below exports data of the ``customers`` space to the ``customers.csv`` file:
+
+.. code-block:: console
+
+    $ tt crud export localhost:3301 customers.csv customers
+
+If the ``customers`` space has five fields (``id``, ``bucket_id``, ``firstname``, ``lastname``, and ``age``), the file with exported data might look like this:
+
+.. code-block:: text
+
+    1,477,Andrew,Fuller,38
+    2,401,Michael,Suyama,46
+    3,2804,Robert,King,33
+    # ...
+
+If a tuple contains a ``null`` value, for example, ``[1, 477, 'Andrew', null, 38]``, it is exported as an empty value:
+
+.. code-block:: text
+
+    1,477,Andrew,,38
+
+
+.. _tt-export-header:
+
+Exporting headers
+-----------------
+
+To export data with a space's field names in the first row, use the ``--header`` option:
+
+.. code-block:: console
+
+    $ tt crud export localhost:3301 customers.csv customers \
+                     --header
+
+In this case, field values start from the second row, for example:
+
+.. code-block:: text
+
+    id,bucket_id,firstname,lastname,age
+    1,477,Andrew,Fuller,38
+    2,401,Michael,Suyama,46
+    3,2804,Robert,King,33
+    # ...
+
+
+.. _tt-export-compound-data:
+
+Exporting compound data
+-----------------------
+
+By default, ``tt`` exports empty values for fields containing compound data like arrays or maps.
+To export compound values in a specific format, use the ``--compound-value-format`` option.
+For example, the command below exports compound values serialized in JSON:
+
+.. code-block:: console
+
+    $ tt crud export localhost:3301 customers.csv customers \
+                     --compound-value-format json
+
+
+.. _tt-export-options:
+
+Options
+-------
+
+..  option:: --batch-queue-size INT
+
+    Specify the maximum number of tuple batches in a queue between a fetch and write threads (the default is ``32``).
```

Comment:
`Specify` and similar werbs seem redundant to me in options that have values. Why not just give the definition:
`-batch-queue-size: the maximum number...`?

I understand that they are optional and setting them _specifies_ some parameter (in contrast to undefined), but the shorter form is pretty common: https://cloud.google.com/sdk/gcloud/reference/compute/disks/update#GCLOUD-WIDE-FLAGS

Reply:
Maybe, will remove in all places.